### PR TITLE
Remove expensive bls key serialization code and use the cached version

### PR DESCRIPTION
**Pull request by rlan35**
Remove the usage of ToLibBLSPublicKey and always use BytesToBLSPublicKey which is the cached version.

Tested locally.

### CODE REVIEW

File: internal/hmyapi/apiv1/blockchain.go [hunk 1]
```diff
@@ -261,8 +263,10 @@ func (s *PublicBlockChainAPI) GetBlockSigners(ctx context.Context, blockNr rpc.B
 		if err != nil {
 			return nil, err
 		}
-		blsPublicKey := new(bls.PublicKey)
-		validator.BLSPublicKey.ToLibBLSPublicKey(blsPublicKey)
+		blsPublicKey := new(bls_core.PublicKey)
```
Comment (gupadhyaya) [inline internal/hmyapi/apiv1/blockchain.go:266]: these allocations are not necessary right?

Comment (rlan35) [reply internal/hmyapi/apiv1/blockchain.go:266]: it is needed if you want to inline the error check. It shouldn't have performance difference as anyway a new pointer needs to be allocated. So I will leave as is.

**Comment by rlan35** [comment]
This PR reduced block sync time for beacon chain from around 0.5s to around 0.25s per block

**Comment by rlan35** [comment]
CPU usage before this PR:
image

After this PR:
image

**Pull request merged**